registry/handlers/referrers.go

```diff
@@ -0,0 +1,364 @@
+package handlers
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"path"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/distribution/distribution/v3"
+	dcontext "github.com/distribution/distribution/v3/context"
+	"github.com/distribution/distribution/v3/manifest/ociartifact"
+	"github.com/distribution/distribution/v3/registry/api/errcode"
+	v2 "github.com/distribution/distribution/v3/registry/api/v2"
+	"github.com/distribution/distribution/v3/registry/storage"
+	"github.com/distribution/distribution/v3/registry/storage/driver"
+	"github.com/gorilla/handlers"
+	"github.com/opencontainers/go-digest"
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+const createAnnotationName = "org.opencontainers.artifact.created"
```

Comment:
@shizhMSFT Is this variable okay? I didn't find any related info about this in the spec.

Reply:
Try searching `org.opencontainers.artifact.created` in https://github.com/opencontainers/distribution-spec/blob/main/spec.md

Reply:
Note: `createAnnotationName` for Artifact manifest and image manfiest are different.

Reply:
Found relevant information in [this page](https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys)